include/irr/asset/ICPUMeshBuffer.h

```diff
@@ -92,6 +100,16 @@ class ICPUMeshBuffer : public IMeshBuffer<ICPUBuffer, ICPUDescriptorSet, ICPURen
     {
         return m_descriptorSet.get();
     }
+	inline void setVertexAttrBuffer(core::smart_refctd_ptr<ICPUBuffer> &buffer, uint16_t attribIndex, E_FORMAT format, uint32_t stride, uint32_t offset)
```

Comment:
you're not actually setting `buffer`

Reply:
@devshgraphicsprogramming should this function even exist?

Reply:
also your function doesnt let setting offset in binding, it'd be better for it to take `SBufferBinding&&`

Reply:
I have already seen that buffer is actaully set before call to that function, so it isn't required, and can't be there

Reply:
resolved

Reply:
you need two functions, one for format+reloffset+etc. and one for buffer binding